src/dlgConnectionProfiles.cpp

```diff
@@ -1126,6 +1135,9 @@ void dlgConnectionProfiles::slot_item_clicked(QListWidgetItem* pItem)
         if (profile_name == QStringLiteral("Clessidra")) {
             val = QStringLiteral("<center><a href='http://www.clessidra.it/'>http://www.clessidra.it</a></center>");
         }
+        if (profile_name == QStringLiteral("Fierymud")) {
+            val = QStringLiteral("<center><a href='http://www.fierymud.org/'>http://www.fierymud.org</a></center>");
```

Comment:
Since the website supports it I recommend you use a secure URI (`https://www.fierymud.org`) instead...